api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -320,6 +324,11 @@ public String getReadOnlyHtml(
   @Override
   public String adminGetReadOnlyHtml(
       String workspaceNamespace, String workspaceName, String notebookName) {
+    if (!NotebookUtils.isJupyterNotebook(notebookName)) {
+      throw new NotImplementedException(
+          String.format("%s type of file is not implemented yet", notebookName));
```

Comment:
```suggestion
          String.format("%s is a type of file that is not yet supported", notebookName));
```

What do you think of this alternative error message?

Reply:
Sounds good!